clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -3565,8 +3565,28 @@ void ItaniumRTTIBuilder::BuildVTablePointer(const Type *Ty) {
   // Check if the alias exists. If it doesn't, then get or create the global.
   if (CGM.getItaniumVTableContext().isRelativeLayout())
     VTable = CGM.getModule().getNamedAlias(VTableName);
-  if (!VTable)
-    VTable = CGM.getModule().getOrInsertGlobal(VTableName, CGM.Int8PtrTy);
+
+  auto VTableTy =
```

Comment:
Please add a comment about address space requirement for SYCL

Reply:
Done, is it enough? [f95e4ae](https://github.com/intel/llvm/pull/7255/commits/f95e4aec16da62d7f1cca0edf2704fc678a62913)

Reply:
I think you can just say -  

To generate valid device code global pointers should have global address space in SYCL.

Reply:
Ok 8b4784dd75de09de28c835c16b8dafca1557e4bf